CMakeLists.txt

```diff
@@ -1502,7 +1502,46 @@ if(ANDROID)
       endif()
     endif()
   endif()
+elseif(OHOS)
+  # disable warnings from the toolchain
+  sdl_compile_options(PRIVATE "-Wno-unused-command-line-argument")
 
+  set(CMAKE_C_COMPILER_EXTERNAL_TOOLCHAIN "")
+  set(CMAKE_CXX_COMPILER_EXTERNAL_TOOLCHAIN "")
```

Comment:
[The CMake documentation says this is for toolchain files only.](https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_EXTERNAL_TOOLCHAIN.html)